astro_2.0/components/SearchInput/SearchInput.tsx

```diff
@@ -213,29 +219,35 @@ export const SearchInput: FC<SearchInputProps> = ({
 
   return (
     <div className={cn(styles.root, className)} ref={ref}>
-      <div className={styles.iconHolder}>
-        <AnimatePresence>
-          {loading ? (
-            <motion.div
-              initial={{ opacity: 0 }}
-              animate={{ opacity: 1 }}
-              exit={{ opacity: 0 }}
-            >
-              <LoadingIndicator className={styles.loader} />
-            </motion.div>
-          ) : (
-            <Icon name="buttonSearch" className={styles.icon} />
-          )}
-        </AnimatePresence>
-      </div>
+      {showLoader ? (
+        <div className={styles.iconHolder}>
+          <AnimatePresence>
+            {loading ? (
+              <motion.div
+                initial={{ opacity: 0 }}
+                animate={{ opacity: 1 }}
+                exit={{ opacity: 0 }}
+              >
+                <LoadingIndicator className={styles.loader} />
+              </motion.div>
+            ) : (
+              <Icon name="buttonSearch" className={styles.icon} />
+            )}
+          </AnimatePresence>
+        </div>
+      ) : null}
       <input
         ref={inputRef}
         tabIndex={0}
         value={value}
         onChange={handleChange}
-        className={cn(styles.input, 'body1')}
+        className={cn(
+          styles.input,
+          { [styles.withoutLoading]: !showLoader },
+          'body1'
```

Comment:
We have a typography `%body1` that can be used in your styles.input like
```
@extend %body1;
```

Reply:
It was there before me. But I will fix it. Thanks.